manager/role_manager.go

```diff
@@ -62,13 +82,29 @@ func (rm *roleManager) Run(ctx context.Context) {
 	if err != nil {
 		log.G(ctx).WithError(err).Error("failed to check nodes for role changes")
 	} else {
+		// Put all members in the removed list first - if the node object exists
+		// for this member, then it is deleted from the removed list.  This is safe
+		// because a node object is created when a node first requests TLS certificates
```

Comment:
>This is safe because a node object is created when a node first requests TLS certificates to be able to communicate with other raft members, so the node object

Sorry but I don't really follow what this means. Maybe clarify why we're doing this ?